templates/boxMenu.jsx

```diff
@@ -0,0 +1,142 @@
+
+  return (
+    <>
+      {backgroundImages &&
+      <div
+        className="background"
+        aria-hidden="true"
+        style={{
+          backgroundImage: 'url(' + backgroundImage + ')',
+          backgroundRepeat: styles._backgroundRepeat,
+          backgroundSize: styles._backgroundSize,
+          backgroundPosition: styles._backgroundPosition
+        }}
+      />
+      }
+
+      <div className="menu__inner boxmenu__inner">
+
+        {(displayTitle || subtitle || body || instruction) &&
+        <div
+          className={classes([
+            'menu__header',
+            'boxmenu__header',
+            headerBackgroundImage && 'has-bg-image',
+            headerMinimumHeight && 'has-min-height'
+          ])}
+          style={ headerMinimumHeight ? { minHeight: headerMinimumHeight + 'px' } : null }
+        >
+
+          {headerBackgroundImages &&
+          <div
+            className="background"
+            aria-hidden="true"
+            style={{
+              backgroundImage: 'url(' + headerBackgroundImage + ')',
+              backgroundRepeat: headerBackgroundStyles._backgroundRepeat,
+              backgroundSize: headerBackgroundStyles._backgroundSize,
+              backgroundPosition: headerBackgroundStyles._backgroundPosition
+            }}
+          />
+          }
+
+          <div className="menu__header-inner boxmenu__header-inner">
+
+            {_graphic?._src &&
+            <div className="menu__image-container boxmenu__image-container">
+              <img
+                className="menu__image boxmenu__image"
+                src={_graphic?._src}
+                alt={_graphic?.alt}
+                aria-hidden={!_graphic?.alt ? true : null}
+              />
+            </div>
+            }
+
+            <div className="menu__header-content">
+
+              {displayTitle &&
+              <div className="menu__title boxmenu__title">
+                <div className={classes([
+                  'menu__title-inner',
+                  'boxmenu__title-inner',
+                  'js-heading'
+                ])} />
+              </div>
+              }
+
+              {subtitle &&
+              <div className="menu__subtitle boxmenu__subtitle">
+                <div className="menu__subtitle-inner boxmenu__subtitle-inner" dangerouslySetInnerHTML={{ __html: compile(subtitle) }}/>
+              </div>
+              }
+
+              {(body || pageBody) &&
+              <div className="menu__body boxmenu__body">
+                <div className="menu__body-inner boxmenu__body-inner" dangerouslySetInnerHTML={{ __html: compile(pageBody || body) }} />
+              </div>
+              }
+
+              {instruction &&
+              <div className="menu__instruction boxmenu__instruction">
+                <div className="menu__instruction-inner boxmenu__instruction-inner" dangerouslySetInnerHTML={{ __html: compile(instruction) }}/>
```

Comment:
Perhaps splitting the attributes across multiple lines helps with readability? 
```suggestion
                <div 
                className="menu__instruction-inner boxmenu__instruction-inner" 
                dangerouslySetInnerHTML={{ __html: compile(instruction) }}
              />
```